lib/celluloid/groups/proactor.rb

```diff
@@ -0,0 +1,56 @@
+require 'thread'
+
+module Celluloid
+  class Group
+    class Proactor < Group
```

Comment:
I don't get how "creating a new thread" relates meaningfully to the Proactor Pattern